### ISSUE TITLE
How to use with Picture image tag?

### ISSUE THREAD

**Issue by tomasts248** [jjcosgrove/jquery-aniview#23]
Basically I tried two ways first adding the class "aniview" to the image fallback like this, but the problem is that it doesn't make use of the source sizes.
```
<picture >
             <source media="(max-width:1369px)" data-srcset="<?php bloginfo('template_url');?>/img/servicios-01-disenopesonalizadov10-x1020.png">
              <source media="(min-width:1370px)" data-srcset="<?php bloginfo('template_url');?>/img/servicios-01-disenopesonalizadov10-x1526.png">              
              <!-- Fallback 2x tamaño -->   
              <img style="max-height: 494px; filter: drop-shadow(0px 0px 41px #e6e6e6);" class="lazyload imagen-servicios mb-5-custom mr-n4 mr-lg-n5 mr-xl-n5 aniview" data-av-animation="customZoomIn" data-src="<?php bloginfo('template_url');?>/img/servicios-01-disenopesonalizadov10-x1526.png" alt="Diseños web Personalizados"> 
           </picture>
```
The other option is to set aniview class directly in the picture tag like this
`<picture class="aniview" data-av-animation="customZoomIn">`
but then this causes weird behaviors, for example drop-shadow property doesn't that quite work well.

**Comment by jjcosgrove** [comment]
Realise this is late/old but will answer anyway...

If I understand correctly, I would think you might have better success wrapping it in a div and applying the class(es) to that div. Then the image will animate irrespective of the breakpoints (not as a user resizes the browser though as that is out of scope of the library and could be handled via custom code)

https://jsfiddle.net/amfjrctL/

As for the lazy load... assuming the lazyload triggers before the `animateThreshold` so that its 'ready' that should also work

https://jsfiddle.net/xvkmL8p4/1/

Finally given that vanilla-lazyload for e.g allows you fire off code on certain events, you could load the images and only once loaded, initialize AniView and it should then fire of animations on those items in view.
